libntxm/common/source/sample.cpp

```diff
@@ -460,6 +467,30 @@ u8 Sample::getPanning(void)
 	return panning;
 }
 
+void Sample::setBasePanning(void)
+{
+	base_panning = panning;
+}
+
+u8 Sample::getBasePanning(void)
+{
+	return base_panning;
+}
+
+#if defined(ARM7)
+
+void Sample::updatePanning(u8 channel)
+{
+  //Ugggghhhhhhhhhhh fix
```

Comment:
This would make more sense as a TODO or FIXME comment, just so it's easier to find in the future.